.github/workflows/cmake.yml

```diff
@@ -0,0 +1,45 @@
+name: CMake
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally well on Windows or Mac.
+    # You can convert this to a matrix build if you need cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install toolchain
+      run: |
+        sudo apt install srecord
+        pushd /tmp
+        wget https://github.com/lowRISC/lowrisc-toolchains/releases/download/20230427-1/lowrisc-toolchain-gcc-rv32imcb-20230427-1.tar.xz
+        tar -xvf lowrisc-toolchain-gcc-rv32imcb-20230427-1.tar.xz
+
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      run: |
+        export PATH=$PATH:/tmp/lowrisc-toolchain-gcc-rv32imcb-20230427-1/bin
+        mkdir sw/c/build
+        cd sw/c/build
+        echo $PATH
```

Comment:
This seems to be leftover from debugging?

Reply:
yeah, removed